feature-interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/TabContent.kt

```diff
@@ -40,7 +41,8 @@ fun TopicsTabContent(
     modifier: Modifier = Modifier
 ) {
     LazyColumn(
-        modifier = modifier.padding(horizontal = 16.dp)
+        modifier = modifier.padding(horizontal = 16.dp),
+        contentPadding = PaddingValues(vertical = 8.dp)
```

Comment:
This does something ever so slightly different, since this is using `vertical` instead of `top`. Can we keep the old spacing amount, but looking much better on the top?